base_layer/core/src/blocks/block.rs

```diff
@@ -24,6 +24,7 @@
 // Version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0.
 use crate::{
     blocks::BlockHeader,
+    chain_storage::BlockchainBackend,
```

Comment:
This dep is unfortunate.

Ideally we should keep Block as low level as we can.
In a perfect world, there's no dep on ConsensusManager either.